src/xcode/ENA/ENA/Source/Extensions/UIStackView+Background.swift

```diff
@@ -0,0 +1,14 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+extension UIStackView {
+	func add(backgroundColor: UIColor) {
+		let viewWithBackgroundColor = UIView(frame: bounds)
+		viewWithBackgroundColor.backgroundColor = backgroundColor
+		viewWithBackgroundColor.autoresizingMask = [.flexibleWidth, .flexibleHeight]
```

Comment:
`autoresizingMask` What year is it? 
Please use Autolayout

Reply:
done.

Reply:
But what happens now if the Stackview gets resized? This would leave the Background in the old size?

Reply:
Added constraints.